packages/edit-site/src/components/style-book/index.js

```diff
@@ -70,10 +70,15 @@ function getExamples() {
 	};
 
 	const otherExamples = getBlockTypes()
-		.filter(
-			( blockType ) =>
-				blockType.name !== 'core/heading' && !! blockType.example
-		)
+		.filter( ( blockType ) => {
+			const { name, example, supports, parent } = blockType;
+			return (
+				name !== 'core/heading' &&
+				!! example &&
+				supports.inserter !== false &&
+				! parent
```

Comment:
If we go this way, I guess we should also exclude those with `ancestor`.

Reply:
Makes sense!
I have also considered the approach described in [this comment](https://github.com/WordPress/gutenberg/issues/47440#issuecomment-1405127584), which is to leave the current logic in place and simply remove the `example` property of the block that we do not want to be listed in the stylebook.
Do you think this PR makes sense?

Reply:
I have found that there are some core blocks that will not show up if I exclude blocks with `parent` and `ancestor`. I added only `"inserter":false` as a condition to keep the current list of blocks.